packages/cache/src/internal/tar.ts

```diff
@@ -14,14 +14,20 @@ async function getTarPath(
   switch (process.platform) {
     case 'win32': {
       const systemTar = `${process.env['windir']}\\System32\\tar.exe`
-      if (compressionMethod !== CompressionMethod.Gzip) {
+      const gnuTar = `${process.env['PROGRAMFILES']}\\Git\\usr\\bin\\tar.exe`
+      if (existsSync(gnuTar)) {
+        // Use GNUtar as default on windows
+        args.push('--force-local')
+        return gnuTar
+      } else if (
+        compressionMethod !== CompressionMethod.Gzip ||
+        (await utils.isGnuTarInstalled())
+      ) {
         // We only use zstandard compression on windows when gnu tar is installed due to
         // a bug with compressing large files with bsdtar + zstd
         args.push('--force-local')
       } else if (existsSync(systemTar)) {
```

Comment:
Nitpicking here. Can simplify the logic to either use `multiple ifs with returns` or use `if-else-if` logic. Having both make readability a bit confusing.

Reply:
Sure can do

Reply:
Would this reduce the speed of the code?

Reply:
Done

Reply:
I don't think it would, and will not be significant either way. It would make it easier to follow though IMO.